### PR TITLE
gNMI-1.3: export only BGP routes to have accurate check in ATE

### PR DESCRIPTION
**Pull request by bkreddy143**
Modified export only inject routes from ATE so that we will have accurate check in ATE on RX side.

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

**Comment by cfernanz** [comment]
opened internal b/294292378 to validate this PR

File: feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func setMED(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root) {
 		t.Fatal(err)
 	}
 	actions5 := stmt.GetOrCreateActions()
+	conditions := stmt.GetOrCreateConditions()
```
Comment (cfernanz) [inline feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go:74]: Let's make this flag controlled/deviation based

**Comment by LimeHat** [comment]
Superseded by #1946

**Pull request closed without merging**